### DISCUSSION TITLE
Can drivers/modules be installed into the docker container for Frigate?

### DISCUSSION THREAD

**Discussion by Dvalin21** [blakeblackshear/frigate#4480] [Ask A Question] [answered]
I'm sure someone has thought of this before, but I wanted to ask as I don't know if the devs will include the driver/module I've requested.  To give some back ground, I'm using [Xpenology/Synology](https://xpenology.com/forum/topic/53817-tinycore-redpill-loader-tcrp/). The kernel for Synology NAS units have been compiled to work on third party hardware. I'm currently running it on an old Dell Poweredge T320 server, 40gb of RAM, 7.5tb of storage on a SHR raid, and a installed AMD HD 6450 1gb GPU. Motherboard picks it up no problem, however the software doesn't. I've requested for this to be added to the kernel, but no word or really a response to it being added. I've installed Frigate NVR via docker-compose (through portainer). How possible is it to install the drive to the container and if so how in the world would I be able to do it? Thanks in advance.

**Comment by NickM-27** [comment, marked as answer]
If it's for the AMD HD 6450 then the driver should already be in the container, however the driver MUST also be on the host otherwise there is no connection for the container to reach the actual hardware

**Reply by Dvalin21** [reply]
Thank you. I will have to figure a way to get it installed.
